Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
Relatively speaking, large PRs indeed are inevitable in some cases.

More so with things like upgrading wx, than this RSP stuff I am doing.  Large PRs would not be necessary here with the RSP if I had a "Find All References" kind of feature here on GNU, like Visual Studio for Windows has.  Without that, I have no way of preventing WIN32 build errors without doing a Ctrl+F find and search through tens of the source files looking for multiple externs or repeated declarations, or waiting for you to post a list of the errors (which you did) and fixing some of those.  Otherwise it's kind of infeasible for me...though I talked to project64 and he said he will fix the WIN32 stuff tomorrow.